docs/starters.yml

```diff
@@ -3747,3 +3747,16 @@
     - Support for Netlify deployment
     - SEO
     - Prettier code styling
+- url: https://johnjkerr.github.io/gatsby-creative/
```

Comment:
@JohnJKerr it seems like your preview is not a gatsby site.

Reply:
Hi @wardpeet - How do you mean? It was created from the gatsby-starter-default, and I took the markup from an existing template and converted it to work with gatsby? Then I deployed it to GitHub Pages via GitHub Actions. Is there something I'm missing?

Reply:
I'm looking at https://blackrockdigital.github.io/startbootstrap-creative/ it's not showing that it's a gatsby site. The html is not the same as what a gatsby site will provide. I don't see react being used, .... Are you sure it's the correct upload?

Reply:
That's the source markup - my preview is over at https://johnjkerr.github.io/gatsby-creative/

Reply:
My bad for linking to the source in the PR description - the URL in the starters.yml is the correct one

Reply:
oh i'm sorry, I clicked on the other link. Thank you!